api/checks/webhook.go

```diff
@@ -84,6 +87,12 @@ func handleCheckSuiteEvent(ctx context.Context, payload []byte) (bool, error) {
 		return false, err
 	}
 
+	appID := checkSuite.GetCheckSuite().GetApp().GetID()
```

Comment:
Actually, one question: what if `App` is nil? Will this be deferencing a null pointer?

Reply:
Designed to avoid nil checks:

`// GetApp returns the App field.
func (c *CheckRun) GetApp() *App {
	if c == nil {
		return nil
	}
	return c.App
}`